Explorer/Assets/DCL/LOD/Components/SceneLODInfoDebug.cs

```diff
@@ -0,0 +1,108 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+using Utility;
+using Object = UnityEngine.Object;
+
+namespace DCL.LOD
+{
+    public struct SceneLODInfoDebugContent
+    {
+        public Color[] OriginalColors;
+        public Renderer[] Renderers;
+        public GameObject[] FaillingCubesGameObjects;
+    }
+
+    public struct SceneLODInfoDebug : IDisposable
+    {
+        private Dictionary<byte, SceneLODInfoDebugContent> SceneLODInfoDebugContents;
+        private Transform MissingSceneParent;
+        public byte CurrentLODLevel;
+
+        private void UpdateContent(SceneLODInfoDebugContent newContent, Color debugColor)
+        {
+            for (int i = 0; i < newContent.Renderers.Length; i++)
+            for (int j = 0; j < newContent.Renderers[i].materials.Length; j++)
+                newContent.Renderers[i].materials[j].color = debugColor;
```

Comment:
every time you access `materials` you get a new array which is a waste of allocations